server.js

```diff
@@ -19,6 +19,9 @@ var users = {};
 var numUsers = 0;
 var status = ("surv", "zomb", "dead");
 
+//30s disconnect timeout
+var dcTimeout = 30;
```

Comment:
Use capitalisation (DISCONNECT_TIMEOUT) for constants. There is a const modifier in javascript but some browsers don't support it.